frontend/taipy-gui/src/components/Taipy/FileDownload.tsx

```diff
@@ -93,11 +93,11 @@ const FileDownload = (props: FileDownloadProps) => {
     const aProps = useMemo(() => (bypassPreview ? {} : { target: "_blank", rel: "noreferrer" }), [bypassPreview]);
 
     return render ? (
-        <label htmlFor={linkId} className={className}>
```

Comment:
I'm afraid that moving `className` to the button will cause breaking change for the current users of Taipy.
The point of having the `className` at the top-level of the component tree of a Taipy element is to facilitate the style customization of any of the component in the element's tree.